iterations_runners/Makefile

```diff
@@ -0,0 +1,26 @@
+HERE != pwd
```

Comment:
Can we use a better variable name? Or, preferably, simpy use `pwd` instead of `${HERE}` later?